### PR TITLE
Filling up sample applications for all iaases

### PR DESCRIPTION
**Pull request by Vishanth**
Filling up the sample applications

### CODE REVIEW

**Comment by Vishanth** [comment]
Thanks Dakshika. I have now corrected 'upperLimit' to 'Threshold'

File: samples/dependency-scaling/sample-groups/openstack/artifacts/php.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                    "value": "vishanth-key"
+                },
+                {
+                    "name": "securityGroups",
+                    "value": "default"
+                }
+            ]
+        }
+    ],
+    "loadBalancer": {
```
Comment (imesh) [inline samples/dependency-scaling/sample-groups/openstack/artifacts/php.json]: This load balancer configuration is not used any more. Shall we remove this  please?
Thanks

Comment (Vishanth) [inline samples/dependency-scaling/sample-groups/openstack/artifacts/php.json]: Removed Load Balancer configurations in all the samples

**Comment by Vishanth** [comment]
Completed the kubernetes sample applications

**Comment by imesh** [comment]
Thanks for the contribution Vishanth. I have now merged this pull request to master branch.

**Comment by Vishanth** [comment]
Thank you Imesh.

**Pull request closed without merging**